practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    data = torch.load(f"data.pt")
</pre><p>Are they supposed to download this data themselves?</p><p>Remember that this notebook needs to be able to run locally.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>